### ISSUE TITLE
Multicast Issue

### ISSUE THREAD

**Issue by ash47** [ash47/LegendsOfDota#233]
http://steamcommunity.com/workshop/filedetails/discussion/296590332/371918937278349496/

> So I was messing around with builds in customs and I picked 6 slots, 2 ults. I thought it would be funny to take Multicast, Death Pulse, Meat Hook, Witchcraft, Essence Aura, and Elder Titan Ultimate (I forget what that one's called). The idea was multicast ET ult and one shot the enemy team, but if that failed with Ahgs I could hook and Death Pulse every second with infinite mana.
> 
> Got an ahgs, got Octarine for more ults, and then when I got Multicast maxed out something unexpected happened, it broke the cooldown of Multicast, removing it entirely. All of a sudden I could 4x multicast Death Pulse 3 casts in a row if I was lucky. So I did the reasonable thing and grabbed Aether lens so I could Death Pulse constantly at longer range and for more damage and Blademail just to dare them to try killing me.
> 
> Tl;Dr Multicast cooldown can be removed by rank 3 + Witchcraft and Octarine, even though doing the math it should still have a cooldown of 3.5 seconds if the CDR stacked favorably, which the tooltip says it doesn't, would be a cooldown of 4.5 with the flat being removed first.

**Comment by testiclesbomber** [comment]
Yes, multicast always broke with no cd
